packetbeat/docs/flows.asciidoc

```diff
@@ -0,0 +1,104 @@
+[[flows]]
+== Configuring Flows to Monitor Network Traffic
+
+You can configure Packetbeat to collect and report statistics on network flows.
+A _flow_ is a group of packets sent over the same time period that share
+common properties, such as the same source and destination address and protocol.
+You can use this feature to analyze network traffic over specific protocols on
+your network.
+
+For each flow, Packetbeat reports the number of packets and the total number of
+bytes sent from the source to the destination. Each flow event also contains
+information about the source and destination hosts, such as their IP address and
+location. For bi-directional flows, Packetbeat reports statistics for the reverse
```

Comment:
@andrewkroh I've created a PR to track this work because I don't have time to do it right now. https://github.com/elastic/beats/issues/4238